### PR TITLE
Add Vietnamese Translator file

### PR DESCRIPTION
**Pull request by mrtungdev**

### CODE REVIEW

File: languages/tgmpa-vi_VN.po [hunk 1]
```diff
@@ -0,0 +1,469 @@
+msgid ""
+msgstr ""
+"Plural-Forms: nplurals=1; plural=0;\n"
+"Project-Id-Version: TGM Plugin Activation by BBLand\n"
```
Comment (GaryJones) [inline languages/tgmpa-vi_VN.po]: Please fix this to be "TGM Plugin Activation 2.6.1" (or whatever version this is for).

File: languages/tgmpa-vi_VN.po [hunk 2]
```diff
@@ -0,0 +1,469 @@
...
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.0.2\n"
+"X-Poedit-Basepath: ..\n"
+"X-Poedit-WPHeader: class-tgm-plugin-activation.php\n"
+"X-Poedit-SourceCharset: UTF-8\n"
+"X-Poedit-KeywordsList: __;_e;_n:1,2;_x:1,2c;_ex:1,2c;_nx:4c,1,2;esc_attr__;"
+"esc_attr_e;esc_attr_x:1,2c;esc_html__;esc_html_e;esc_html_x:1,2c;_n_noop:1,2;"
+"_nx_noop:3c,1,2;__ngettext_noop:1,2\n"
+"Last-Translator: \n"
```
Comment (GaryJones) [inline languages/tgmpa-vi_VN.po]: Please fill with your name (if you were the last translator).

**Review by GaryJones — CHANGES_REQUESTED** [review]
Thanks @WeAreUnique - just a couple of headings to tweak please.

File: languages/tgmpa-vi_VN.po (hunk 1 above)
Comment (mrtungdev) [reply languages/tgmpa-vi_VN.po]: @GaryJones Updated :)

File: languages/tgmpa-vi_VN.po (hunk 2 above)
Comment (mrtungdev) [reply languages/tgmpa-vi_VN.po]: @GaryJones Updated :)

File: languages/tgmpa-vi_VN.po [hunk 3]
```diff
@@ -1,7 +1,9 @@
+# Translation of TGM Plugin Activation in Vietnamese (VN)
+# This file is distributed under the same license as the TGMPA package.
 msgid ""
 msgstr ""
 "Plural-Forms: nplurals=1; plural=0;\n"
-"Project-Id-Version: TGM Plugin Activation by BBLand\n"
+"Project-Id-Version: TGM Plugin Activation v2.6.1 by BBLand.net\n"
```
Comment (GaryJones) [inline languages/tgmpa-vi_VN.po]: Please remove the "by BBLand.net" bit. You didn't create TGMPA; your credits for translating are in the other headers.

File: languages/tgmpa-vi_VN.po [hunk 4]
```diff
@@ -15,7 +17,7 @@ msgstr ""
 "X-Poedit-KeywordsList: __;_e;_n:1,2;_x:1,2c;_ex:1,2c;_nx:4c,1,2;esc_attr__;"
 "esc_attr_e;esc_attr_x:1,2c;esc_html__;esc_html_e;esc_html_x:1,2c;_n_noop:1,2;"
 "_nx_noop:3c,1,2;__ngettext_noop:1,2\n"
-"Last-Translator: \n"
+"Last-Translator: BBLand.NET\n"
```
Comment (GaryJones) [inline languages/tgmpa-vi_VN.po]: The expectation here is the name and email address of the individual, not the team. See https://www.gnu.org/software/gettext/manual/html_node/Header-Entry.html

File: languages/tgmpa-vi_VN.po (hunk 3 above)
Comment (mrtungdev) [reply languages/tgmpa-vi_VN.po]: Updated 👍

**Review by GaryJones — APPROVED** [review]

**Pull request merged**